src/App.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/home';
+import Navbar from './components/navigation';
+import Quotes from './components/quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/200793103-95856bd0-2e82-48c9-a0aa-56c2275a326b.png)

- [ ] This Getting Started section should have those commands which are mandatory to run to use your application :blush: to make your readme more descriptive :+1:

```
git clone git@github.com:jaamanu/maths-magician.git
npm install
npm start
```